docs/dev-guide/iframe.md

```diff
@@ -188,6 +188,11 @@ const api = new JitsiMeetExternalAPI(domain, options);
 ```
 **Note:** **`TILE_VIEW_MAX_COLUMNS`** accepts values from 1 to 5. The default value is 5.
 
+
+:::warning
```

Comment:
Shouldn't we have this at the top of the file?

Reply:
Well, I don't know about that. The thing is that this doesn't apply to the sections above where it is now

Reply:
Ah right. Let's leave it there then!